activiti-core/activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/impl/TaskRuntimeImpl.java

```diff
@@ -262,8 +262,6 @@ public Task create(CreateTaskPayload createTaskPayload) {
         task.setFormKey(createTaskPayload.getFormKey());
         task.setOwner(securityManager.getAuthenticatedUserId());
         taskService.saveTask(task);
-        taskService.addCandidateUser(task.getId(),
```

Comment:
With this change, the newly created task will be available only for the admin with no candidate user/group is provided in the payload. Is that the expected behavior?

Reply:
As far as I know yes. The front-end check that there is at least an assignee or a candidate user o group.
We can improve this code doing an additional check: if there is an assignee or a candidate user/group, nothing will happen, otherwise, we can set the current user as candidate user. 
@mauriziovitale what do you think?

Reply:
Every time we create a standalone task we provide the assignee. So the task will be available for him and because there are no candidate that user should not be allowed to release a task. @erdemedeiros

Reply:
Currently, there is no check at the API level, but it should be ok for now.